WordPress/Classes/ViewRelated/Stats/Insights/Two Column Stats/TwoColumnCell.swift

```diff
@@ -85,7 +85,7 @@ private extension TwoColumnCell {
     func toggleViewMore() {
         let showViewMore = !dataRows.isEmpty && statSection == .insightsAnnualSiteStats
         viewMoreView.isHidden = !showViewMore
-        rowsStackViewBottomConstraint.constant = showViewMore ? viewMoreHeightConstraint.constant : 0
+        rowsStackViewBottomConstraint?.isActive = showViewMore
```

Comment:
Since the viewMoreHeightConstraint has been removed, I believe you can remove the IBOutlet reference on line 14?